modules/hitlnext/src/backend/service.ts

```diff
@@ -0,0 +1,100 @@
+import * as sdk from 'botpress/sdk'
+import _ from 'lodash'
+
+import { Config } from '../config'
+import { MODULE_NAME } from '../constants'
+import { ExitTypes, IHandoff } from '../types'
+import { StateType } from '.'
+import Repository from './repository'
+
+export const toBpDestination = (botId: string, handoff: Pick<IHandoff, 'userId' | 'userThreadId' | 'userChannel'>) => {
+  return { botId, target: handoff.userId, threadId: handoff.userThreadId, channel: handoff.userChannel }
+}
+
+class Service {
+  constructor(private bp: typeof sdk, private state: StateType, private repository: Repository, private realtime) {}
+
+  async createHandoff(
+    botId: string,
+    dest: Pick<IHandoff, 'userId' | 'userThreadId' | 'userChannel' | 'status'>,
+    timeoutDelay: number
+  ) {
```

Comment:
I know but one of the advantages of having explicit typings is that it defines expected typings. For example, if you change the return statement it will type-check without error even though the typing is not the one you expect the function to return.